config.rb

```diff
@@ -129,13 +129,7 @@
 page '*.json'
 
 Middleman::Docsite.projects.each do |project|
-  proxy(
-    "/gems/#{project.name}/index.html",
-    '/gem-index-redirect.html',
-    locals: { path: project.latest_path },
-    layout: false,
-    ignore: true
-  )
+  redirect "gems/#{project.name}/index.html", to: project.latest_path
```

Comment:
A bit of an off-topic, but I'd love some discussion and input

I was wondering if we could configure redirects for the rest of the pages, like this one:
https://dry-rb.org/gems/dry-monads/do-notation/
Right now it shows 404, but it would be great to have some kind of redirects for the rest of the pages

Reply:
Just what I'm trying to do right now =)

Reply:
@Morozzzko here is a working example:

```ruby
  source_path = "./source#{project.latest_path}/"
  Dir["#{source_path}**/*"].each do |path|
    file = path.gsub(source_path, '').gsub('.html.md', '.html')
    redirect "gems/#{project.name}/#{file}", to: "#{project.latest_path}/#{file}"
  end
```

This approach has a disadvantage – changing the structure of the docs will result in a redirect to 404.

Reply:
Well, not if we use `oldest_path` instead of `latest_path`. Or configure some other way.

Anyway, it's good as is. Solves 90% of cases now

Reply:
> Well, not if we use `oldest_path` instead of `latest_path`. Or configure some other way.

In some cases `oldest_path` will redirect to 0.x instead of 1.x versions.

Reply:
I... didn't think about that 🤔 

Well, that's the general problem with the version switcher. We'll figure something out eventually

Reply:
> In some cases oldest_path will redirect to 0.x instead of 1.x versions ¯\(ツ)/¯

how come? 😕